test/Interop/Cxx/stdlib/msvcprt-module-interface.swift

```diff
@@ -0,0 +1,23 @@
+// RUN: %empty-directory(%t)
```

Comment:
Is this the test from @compnerd's patch?

Reply:
Yes just using for testing on ci. I can drop it